src/styles/blocks/header.scss

```diff
@@ -38,5 +40,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $transition;
```

Comment:
set properties which transition applied for here as well

Reply:
it is important for performance. [here is explanation](https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x)

Reply:
> it is important for performance. [here is explanation](https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x)

thanks for the explanation)